src/xform_to_datalog/datalog_facts_test.cc

```diff
@@ -154,9 +154,12 @@ says_hasTag("particle", "recipe.particle.out", owner, "tag") :- ownsAccessPath(o
 
 
 // Authorization Logic
-.decl grounded_dummy(x0: symbol)
-.decl says_cond1(x0: symbol, x1: symbol)
-.decl says_fact1(x0: symbol, x1: symbol)
+.type SomeType <: symbol
```

Comment:
@bgogul Is there any way we can make the tests in `xform_to_datalog` any less brittle? I think I'll have to change all of these tests every time the authorization logic backend is changed.

Reply:
There might not be... and even if there is maybe it isn't worth the effort. It might just be that the next PR is the last one where the backed for the rust implementation changes.